### PR TITLE
SDT-129: add commisioning module

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: interceptors/.springBeans
Comment (paulridout) [inline interceptors/.springBeans:1]: Is this file needed?  Looks to be an old Eclipse Spring config file (see https://stackoverflow.com/questions/28572451/what-is-the-springbeans-file).

File: interceptors/src/main/java/uk/gov/moj/sdt/interceptors/service/IPersistServiceRequest.java
Comment (paulridout) [inline interceptors/src/main/java/uk/gov/moj/sdt/interceptors/service/IPersistServiceRequest.java:1]: I can't find any classes that implement this interface - am I overlooking something?

File: interceptors/src/main/java/uk/gov/moj/sdt/interceptors/enricher/BulkFeedbackEnricher.java [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+import java.util.regex.Pattern;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import uk.gov.moj.sdt.utils.SdtContext;
+
+import static uk.gov.moj.sdt.domain.api.IIndividualRequest.IndividualRequestStatus.REJECTED;
+
+
```
Comment (paulridout) [inline interceptors/src/main/java/uk/gov/moj/sdt/interceptors/enricher/BulkFeedbackEnricher.java:46]: Remove one of these blank lines.

File: interceptors/src/main/java/uk/gov/moj/sdt/interceptors/service/IPersistServiceRequest.java
Comment (kiran-yenigala-hmcts) [reply interceptors/src/main/java/uk/gov/moj/sdt/interceptors/service/IPersistServiceRequest.java:1]: Yes, not done this intentionally, as the implementation will be provided in civil-sdt and civil-sdt-commissioning

File: interceptors/src/main/java/uk/gov/moj/sdt/interceptors/enricher/SubmitQueryEnricher.java [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+     */
+    @SuppressWarnings({"java:S5852", "java:S5857"})
+    private String loadSummaryResults(String newXml) {
+
+        // Get the system specific response from thread local to inject into the outbound message
+        String outXml = newXml;
+
+        Pattern resultsPattern = Pattern.compile("<(qresp|ns2)\\:results\\/>|<(qresp|ns2)\\:results.*\\>.*<\\/(qresp|ns2)\\:results\\>");
+//        String replacementXml = SdtContext.getContext().getClaimDefencesSummaryResultsXml();
+        String replacementXml = "";
```
[the remark this answers is not in the document]
Comment (kiran-yenigala-hmcts) [reply interceptors/src/main/java/uk/gov/moj/sdt/interceptors/enricher/SubmitQueryEnricher.java:138]: yes, it will be removed after merge into master

File: interceptors/.springBeans
Comment (kiran-yenigala-hmcts) [reply interceptors/.springBeans:1]: removed
